classes/ObjectModel.php

```diff
@@ -1104,7 +1104,7 @@ public static function displayFieldName($field, $class = __CLASS__, $htmlentitie
         }
 
         $key = $class.'_'.md5($field);
-        return ((is_array($_FIELDS) && array_key_exists($key, $_FIELDS)) ? ($htmlentities ? htmlentities($_FIELDS[$key], ENT_QUOTES, 'utf-8') : $_FIELDS[$key]) : $field);
+        return ((is_array($_FIELDS) && property_exists($_FIELDS, $key)) ? ($htmlentities ? htmlentities($_FIELDS[$key], ENT_QUOTES, 'utf-8') : $_FIELDS[$key]) : $field);
```

Comment:
@wdammak 
I forgot to mark one change. $_FIELDS is an array. cannot use property_exists.
Please remove this change also.

Thanks.

Reply:
done